app/src/main/res/values/string-untranslated.xml

```diff
@@ -24,4 +24,13 @@
     <string name="useOurAppNotificationDescription">Here\'s a simple way to reduce its reach.</string>
     <string name="useOurAppShortcutAddedText">Success! %s has been added to your home screen.</string>
     <string name="useOurAppDeletionDialogText">Checking your feed in DuckDuckGo is a great alternative to using the Facebook app!&lt;br/&gt;&lt;br/&gt;But if the Facebook app is on your phone, it can make requests for data even when you\'re not using it.&lt;br/&gt;&lt;br/&gt;Prevent this by deleting it now!</string>
+
+    <!-- Fire Animation settings -->
+    <string name="settingsFireAnimation">Fire Button Animation</string>
```

Comment:
In theory, if this is not an experiment we should be getting translations for all of these strings before merging.

Reply:
ha! that's good to know. So we will need translations before merging...good to know.

Going to move them to `strings.xml` but then I expect bitrise failing until translations are ready...

Reply:
That's the theory anyway although I probably have not been clear about it when we talked about translations. Feel free to merge and create another PR for translations once you have them if you want to release asap.